Surface_mesh_shortest_path/include/CGAL/Surface_mesh_shortest_path/Surface_mesh_shortest_path.h

```diff
@@ -2739,11 +2810,21 @@ class Surface_mesh_shortest_path
 
   \param vertex A vertex of the input face graph
   */
-  Point_3 point(const vertex_descriptor vertex) const
+  decltype(auto) point(const vertex_descriptor v) const
```

Comment:
When changing the name of the argument, don't forget to change it in the documentation as well.
From documentation build (CGAL-5.6-Ic-22:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-22/include/CGAL/Surface_mesh_shortest_path/Surface_mesh_shortest_path.h:2811: warning: argument 'vertex' of command @param is not found in the argument list of CGAL::Surface_mesh_shortest_path< Traits, VIM, HIM, FIM, VPM >::point(const vertex_descriptor v) const
/home/cgal-testsuite/cgal_doc_build/CGAL-5.6-Ic-22/include/CGAL/Surface_mesh_shortest_path/Surface_mesh_shortest_path.h:2811: warning: The following parameter of CGAL::Surface_mesh_shortest_path::point(const vertex_descriptor v) const is not documented:
  parameter 'v'`

Reply:
resolved by [0e5c2c4](https://github.com/CGAL/cgal/pull/6739/commits/0e5c2c42b9192dd1350641bd6d1c545487ff2ab9)